scripts/run_cnex.py

```diff
@@ -0,0 +1,168 @@
+import os
+import subprocess
+import sys
+
+ExcludeTests = [
+    #"t/array-fraction.neon",        # exception
```

Comment:
All these exclusions that are commented out should be removed entirely. There's no reason to keep the exclusion here if the test passes.

Reply:
Removed.